### DISCUSSION TITLE
How to copy a code block with one click?

### DISCUSSION THREAD

**Discussion by SkDness** [gsantner/markor#2336] [General] [answered]
I would like to ask, how can I copy a code block with one click? If there is no such function, is there any way to achieve it.

**Comment by gsantner** [comment]
there is no such option. Feel free to implement it howerver, or manually select the needed text and copy it by popup.

**Reply by SkDness** [reply]
Just like the pictures show.
![IMG_20240701_222108.jpg](https://github.com/gsantner/markor/assets/77217600/c2200f35-b6b3-4e3a-a1f1-0ee1008f0de3)

**Comment by guanglinn** [comment, marked as answer]
See PR #2345.

**Reply by SkDness** [reply]
Thank You!

**Comment by Benko111** [comment]
I've just discovered the new copy  button for code blocks and had to come here to express my happiness about it. And what can I say, this is just so so great. In the past it was a real hussle to manually copy large code blocks without any means to select them automatically - but since Markor has integrated so deeply into my workflow and totally fits my way of mobile front end development, I had to put up with it. In this sense: thanks so much to whoever came up with that, it's a huge enhancement for me. 👍

**Reply by SkDness** [reply]
New copy button for code blocks is great，thanks to all contributors.
